.github/workflows/holonix-cache.yaml

```diff
@@ -1,92 +1,79 @@
+      - uses: cachix/cachix-action@v15
+        with:
+          name: holochain-ci
+      - name: Cache default Holonix flake on [${{ matrix.branch }}]
         env:
-          NIX_CONFIG: "access-tokens = github.com=${{ secrets.GITHUB_TOKEN }}"
-          CACHIX_AUTH_TOKEN: "${{ secrets.CACHIX_AUTH_TOKEN_HOLOCHAIN_CI }}"
+          # NIX_CONFIG: "access-tokens = github.com=${{ secrets.GITHUB_TOKEN }}"
+          CACHIX_AUTH_TOKEN: "${{ secrets.CACHIX_AUTH_TOKEN }}"
         run: |
-          set -xe
-
-          target=${{ matrix.target }}
+          # Create a workspace and initialise a new default Holonix flake
+          cd $(mktemp -d)
+          nix flake init -t "github:holochain/holonix/${{ matrix.branch }}#holonix-default" # TODO
 
           # See https://docs.cachix.org/pushing#id1
-          nix develop --build -L --profile result-develop \
-            ${{ matrix.extra_args }} \
-            ${{ format(matrix.target, matrix.platform) }}
+          nix develop --build -L --profile result-develop
 
-          nix build -L --profile result-build \
-            ${{ matrix.extra_args }} \
-            ${{ format(matrix.target, matrix.platform) }}
+          nix build -L --profile result-build
 
-          # Don't exit if this fails so we can clean up the profile
+          # Push both results to the cache
           for i in result-*; do
-            cachix push holochain-ci $i || true
+            cachix push holochain-ci $i
           done
-
-          rm result*
   cache-check:
     needs:
       - cache-update
     strategy:
       fail-fast: false
       matrix:
-        target:
-          - "github:holochain/holochain#devShells.{0}.holonix"
-          - "github:holochain/holochain#packages.{0}.hc-scaffold"
-        extra_args:
-          - ""
-          - "--refresh --override-input versions github:holochain/holochain?dir=versions/0_2"
-          - "--refresh --override-input versions github:holochain/holochain?dir=versions/0_2_rc"
-          - "--refresh --override-input versions github:holochain/holochain?dir=versions/0_3_rc"
-          - "--refresh --override-input versions github:holochain/holochain?dir=versions/weekly"
-        platform:
-          - x86_64-darwin
-          - aarch64-darwin
-          - x86_64-linux
+        os: [ ubuntu-latest, macos-latest, macos-13 ]
+        branch: [ main, main-0.2, main-0.3 ]
     runs-on: ubuntu-latest
     steps:
-      - uses: cachix/install-nix-action@v26
-      - uses: cachix/cachix-action@v14
+      - uses: cachix/install-nix-action@v27
+        with:
+          install_url: https://releases.nixos.org/nix/nix-2.22.1/install
+      - uses: cachix/cachix-action@v15
         with:
           name: holochain-ci
       - name: Check the Holonix cache
         uses: holochain/nix-cache-check@v1
-        env:
-          NIX_CONFIG: "access-tokens = github.com=${{ secrets.GITHUB_TOKEN }}"
+        # env:
+        #  NIX_CONFIG: "access-tokens = github.com=${{ secrets.GITHUB_TOKEN }}"
```

Comment:
```suggestion
```

Reply:
We almost definitely need this, I just won't set it up until I see an error. Prefer to remove it anyway and add it back later?

I'm fine either way, just wanted to say that this isn't commented out forever

Reply:
No, let's keep it in that case.